cernopendata/modules/fixtures/data/docs/cms-guide-for-conddb/cms-guide-for-conddb.md

```diff
@@ -2,14 +2,34 @@ This page explains the use of global tags and the condition database with the CM
 
-These records are stored in the condition database. Condition data include non-event-related information (Alignment, Calibration, Temperature, etc.) and parameters for the simulation/reconstruction/analysis software. For 2011-2012 CMS Open Data, the condition data are provided as sqlite files in the `/cvmfs/cms-opendata-conddb.cern.ch/` directory, which is accessible to the CMS Open Data VM.
+These records are stored in the condition database. Condition data include non-event-related information (Alignment, Calibration, Temperature, etc.) and parameters for the simulation/reconstruction/analysis software. For CMS Open Data, the condition data are provided as sqlite files in the `/cvmfs/cms-opendata-conddb.cern.ch/` directory, which is accessible to the CMS Open Data VM.
 
 Most [physics objects](/docs/cms-physics-objects-2011) in the CMS Open Data are already calibrated and ready-to-use, and no additional corrections are needed other than selection and identification criteria, which will be applied in the analysis code. Therefore, simple analyses do not need to access the condition database. Examples of such analyses are [the di-muon spectrum example](/record/5001) or [the Higgs analysis example](/record/5500).
 
 However, access to the condition database is necessary, for example, for jet energy corrections and trigger configuration information. Examples of such analyses are for [the PAT object production](/record/233) or [the top quark pair production](/record/5000).
 
 Note that when you need to access the condition database, the first time you run the job on the CMS Open Data VM, it will download the condition data from the `/cvmfs` area. It will take time (an example run of a 10 Mbps line took 45 mins), but it will only happen once as the files will be cached on your VM. The job will not produce any output during this time, but you can check the ongoing processes with the command 'top' and you can monitor the progress of reading the condition data to the local cache with the command 'df'.
 
+---
+
+**For 2010 collision data**, the global tag available in the  `/cvmfs` area is FT_R_42_V10A. When using the "CMS-OpenData-1.1.2" VM, it is recommended reading the condition data from there. First, set the symbolic links:
+
+```shell
+ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_R_42_V10A FT_R_42_V10A
+ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_R_42_V10A.db FT_R_42_V10A.db
+```
+
+Then, define the correct set of condition data by mentioning the Global Tag in the configuration file of the job.
+
+```shell
+#globaltag
+process.GlobalTag.connect = cms.string('sqlite_file:/cvmfs/cms-opendata-conddb.cern.ch/FT_R_42_V10A.db')
+process.GlobalTag.globaltag = 'FT_R_42_V10A::All'
+```
+
+Note that this only works in the "CMS-OpenData-1.1.2" version of the 2010 CMS Open Data VM.
```

Comment:
Maybe we should put this note in bold font.

Reply:
👍